### DISCUSSION TITLE
Notes

### DISCUSSION THREAD

**Discussion by trbouma** [dgc-cgn/CAS-Digital-Trade-Documentation#7] [General]
The question is - given the diversity of the ecosystem. and the diversity of the standards, what is(are) the best candidate(s) for conformity assessment?

Synthesizing the discussion to date -

- Focus on the MLETR, with emphasis on "legal certainty" defining criteria in relation to: 

- General Reliability

- General Interoperability

Begin to work through functional equivalence to define criteria, choosing one or two to focus on - my recommendation:

- Writing 

- Signature

- Others, as appropriate

Develop a Proposed Statement of Verification/Validation (or certificate)
Using all, or a subset of the above.
The criteria above, need to be as technology-agnostic as possible.

As for technical standards, it is too early to 'standardize' - my recommendation that in the assessing the criteria above, the technical standards used, be cited as evidence in the assessment of the criteria. It is important to realize the intent of the exercise is to enable participants to support a claim others can rely on, versus telling them what they should implement. It's a fine balance, and it is easy to err to the latter.

we can start iterating through the above elements. I am a firm believer in 'less is better, but more if we can manage.' I think we can do: 1) General Reliability, and 2) General Interoperability, and maybe one or two of functional equivalents.

In conclusion, if I were to sum up the value of this exercise to a potential user, I would say this: 

'The conformity assessment scheme enhances the legal effectiveness of digital trade documentation by means participants demonstrating their general reliability and interoperability which is formally indicated by a statement of verification/validation issued by an independent third party.'
